src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -134,38 +139,33 @@ public UUID processAudioRequest(Integer requestId) {
             );
             mediaRequestService.updateAudioRequestStatus(requestId, FAILED);
 
+            if (mediaRequestEntity != null && hearingEntity != null) {
```

Comment:
We shouldn't need to worry about hearingEntity being null here - since MediaRequestEntity has hearing `optional = false` like sql NOT NULL on media_request.hea_id column.
I suppose it's safer to keep like this, should things change in the entities / schema.